torch/_prims/__init__.py

```diff
@@ -875,6 +885,13 @@ def _add_nvfuser(fd: Any, a: TensorLikeType, b: TensorLikeType):
     type_promotion=ELEMENTWISE_PRIM_TYPE_PROMOTION_KIND.DEFAULT,
 )
 
+copysign = _make_elementwise_binary_prim(
```

Comment:
Copysign can be implemented using signbit, I think

Reply:
I saw this specific header file for copysign, which has some special logic for bfloat16 and fp16.
https://github.com/pytorch/pytorch/blob/master/c10/util/copysign.h

Reply:
That's great detail but with default type promotion the kernel will never operator on half or bfloat16 types because it will promote to fp32 instead, anyway

Reply:
I removed the copysign prim, so it is reference-only.